README.md

```diff
@@ -123,9 +123,9 @@ use you should never need or want to edit these.
 
 Now that the script is saved in your Google Drive folder we need to activate it.
 To do so click the menu `Resources->Advanced Google services`.  
-In the popup which will open set "Google Calendar API" to `enabled` (click the
-switch on its row on the right).  
-Once you have done this click on the link which says "Google API Console": you
+In the popup which will open set "Calendar API" to `enabled` (click the
+switch on its row on the right) and press "Okay".  
+Once you have done this open `Resources->Advanced Google services` again and click on the link which says "Google API Console": you
```

Comment:
Great, thank you!

Now there is only one thing to fix: this line is too long. As you can read in the [CONTRIBUTING.md file](https://github.com/GioBonvi/GoogleContactsEventsNotifier/blob/development/.github/CONTRIBUTING.md) Markdown files' styling should follow a certain set of rules: one of these rules is that no line should be longer than 80 characters, so you should break this line in two just before "click".